receiver/elasticsearchreceiver/metadata.yaml

```diff
@@ -7,6 +7,12 @@ resource_attributes:
   elasticsearch.node.name:
     description: The name of the elasticsearch node.
     type: string
+  elasticsearch.endpoint:
+    description: Endpoint of the queried elasticsearch node.
+    type: string
```

Comment:
I did not decide to go for `elasticsearch.node.endpoint`, because we query only one node, but the results can be for many nodes and this would be a misleading hint that both `elasticsearch.node.endpoint` and `elasticsearch.node.name` refer to the same node, while they would not.